libkbfs/tlf_journal.go

```diff
@@ -2046,6 +2064,19 @@ func (j *tlfJournal) putBlockData(
 		return err
 	}
 
+	if err := j.checkInfoFileLocked(); err != nil {
+		return err
+	}
+
+	if j.needInfoFile {
```

Comment:
don't need this block since `checkInfoFileLocked` already does this right?

Reply:
Oops, yeah that was leftover after a refactor.  Removed, thanks!